Makefile

```diff
@@ -129,7 +129,7 @@ casper_files:=recommended_source_epoch-spec.k \
               deposit_exists-failure-spec.k \
               proc_reward-spec.k \
               vote-1-2-3-4-5-6-success-1-spec.k \
-              vote-1-2-3-4-5-6-success-2-spec.k \
+              #vote-1-2-3-4-5-6-success-2-spec.k \
```

Comment:
@denis-bogdanas This does NOT work as you expected. Since you have the backslash in the end, having `#` here means that you commented out all the following lines below. Simply deleting this line, and put a comment in the separate place that you did so.

Reply:
Please make yourself clear that the jenkins log shows that other specs are not generated.